get-started/guide-neo4j-browser/guide-neo4j-browser.adoc

```diff
@@ -8,45 +8,46 @@
 :section: Get Started
 :section-link: get-started
 :experimental:
-:neo4j-version: 2.3.0
+:neo4j-version: 3.3.4
 
 .Goals
 [abstract]
-This article demonstrates how to use the Neo4j browser for querying, visualization and data interaction.
+This article demonstrates how to use the Neo4j browser for querying, visualization, and data interaction.
 
 .Prerequisites
 [abstract]
-Please have link:/download[Neo4j downloaded] and installed. It helps if you read the section on link:/developer/get-started/graph-database[Graph Databases].
+Please have link:/download[Neo4j^] downloaded and installed.
+It also helps if you have read the section on link:/developer/get-started/graph-database[graph databases^].
 
 [role=expertise]
 {level}
 
 toc::[]
 
-:img: .
+//:img: .
 
 == Installing and Starting Neo4j Browser
 
-First, http://neo4j.org/download[download] Neo4j. 
-Use the provided installer, start the Neo4j Desktop App and start the database after (optionally) selecting a location for your database files.
-
-Alternatively unzip the downloaded file, and start Neo4j by following the instructions in the readme (`bin/neo4j start`).
-
+If you haven't already, http://neo4j.org/download[download^] Neo4j. 
+Using the provided instructions (shown when downloading), follow Step 1 to install and start the Neo4j Desktop application.
+Instructions can also be found in the readme (`bin/neo4j start`) after unzipping the downloaded file.
```

Comment:
That's a separate download. Perhaps mention that "Alternatively, after downloading the zipped server-only package". We should probably recommend running `bin/neo4j console` these days and also show the windows equivalent `\bin\neo4j.bat console`  (in a terminal)

Reply:
Ok. That makes sense. I'd need access to a Windows machine before I can run the Windows steps, though. Why should we recommend running bin/neo4j console (i.e. what does that do)?